account_currency_tools/demo/account_move.xml

```diff
@@ -1,329 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<odoo>
-        <record id="res_partner_23" model="res.partner">
```

Comment:
why we delete this file?
it's not used?

Reply:
Yes, it's not used from a long time ago, it was even already commented [when migrated](https://github.com/vauxoo/addons-vauxoo/commit/5fc63ace4842427e3a4b200d2c066cc479846840#diff-19edcbfc7004f63abf1e9b9c1d857562L34)